include/constants/pokemon.h

```diff
@@ -307,6 +307,7 @@
 #define EVO_SCRIPT_TRIGGER_DMG            36     // Pokémon has specified HP below max, then player interacts trigger
 #define EVO_DARK_SCROLL                   37     // interacts with Scroll of Darkness
 #define EVO_WATER_SCROLL                  38     // interacts with Scroll of Waters
+#define EVO_ITEM_DAY                      39     // specified item is used on Pokémon, is day
```

Comment:
I would add `EVO_ITEM_NIGHT` for completeness' sake.

In that same subject, `EVO_ITEM_HOLD` would also be useful for users, as I've seen multiple people ask for it.

Reply:
Sure, why not.